hw/ip/otbn/doc/dv/index.md

```diff
@@ -155,75 +167,107 @@ This paragraph implies eight coverage points (four flags times two values) for t
 
 #### ADD
 
+This instruction uses the `R` encoding schema, with covergroup `enc_r_cg`.
+
 - Cross the three possible signs (negative, zero, positive) for each input operand (giving 9 points).
 
 #### ADDI
 
-As for `ADD`.
+This instruction uses the `I` encoding schema, with covergroup `enc_i_cg`.
+
+Specific points as for `ADD`.
```

Comment:
I don't understand why the are like ADD. For example, you want to look at bit 31 for sign extension for ADDI and ignore it for ADD.

Reply:
What I mean here is "the coverage points that are to do with addition in particular (rather than just "here's an instruction that takes the following operands") are the same for ADDI and ADD. I'll rephrase this, and the other equivalent places, to make it a bit clearer.

Reply:
Hmm. Actually, rephrasing it to be clear is a bit difficult. Instead, let's just do the copy-paste and try to remember to keep everything in sync. See #6722.